UMN-AutoPackager/public/Get-NewPackageFile.ps1

```diff
@@ -0,0 +1,141 @@
+function Get-NewPackageFile {
+<#
+.SYNOPSIS
+Cmdlet that downloads a file from a web server or file share and saves it to the specified location.
+.DESCRIPTION
+Get-NewPackageFile is a function that when provided with a Source file path, UNC, or URI (as a string)
+and a Destination path or UNC (as a string) will copy or download the Source file to the Destination.
+Characters of a space (" ") or an 'escaped' space ("%20") are replaced with an underscore ("_") when
+saved to the Destination.
+.PARAMETER Source
+A string that must be a file path, UNC, or URI of the file to be saved to the Destination location.
+.PARAMETER Destination
+A string that must be a file path or UNC where the Source file will be saved.
+.EXAMPLE
+Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "D:\Temp\File2.txt"
+.EXAMPLE
+Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "D:\Temp\"
+.EXAMPLE
+Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "\\fileserver\Temp\File1.txt"
+.EXAMPLE
+Get-NewPackageFile -Source "\\fileserver\Test\File1.txt" -Destination "D:\Temp\File1.txt"
+.EXAMPLE
+Get-NewPackageFile -Source "\\fileserver1\Test\File1.txt" -Destination "\\fileserver2\Temp\File1.txt"
+.EXAMPLE
+Get-NewPackageFile -Source "https://webserver.xyz/Test/File1.txt" -Destination "C:\Temp\File1.txt"
+.EXAMPLE
+Get-NewPackageFile -Source "https://webserver.xyz/Test/File1.txt" -Destination "\\fileserver\Temp\File1.txt"
+.INPUTS
+String
+.NOTES
+If the Destination parameter is only a file path and does not include a filename, the filename obtained from
+the Source parameter is used during the copy. Requires use of the "Get-RedirectedUri" function.
+#>
+    
+    [CmdletBinding(SupportsShouldProcess)]
```

Comment:
Any reason to not just use [CmdletBinding]()] which would include all the other parameters not just whatif?

Reply:
Reading up on it it looks like it doesn't enable those without adding that. Resolving the conversation.